sdk/storage/Azure.Storage.DataMovement.Files.Shares/src/Azure.Storage.DataMovement.Files.Shares.csproj

```diff
@@ -20,8 +20,8 @@
     <PackageId />
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Storage.Files.Shares" />
-    <!-- <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\Azure.Storage.Files.Shares\src\Azure.Storage.Files.Shares.csproj" />-->
+    <!--<PackageReference Include="Azure.Storage.Files.Shares" />-->
```

Comment:
We should probably create an issue that tracks changing this back before release.   But I'll defer to @seanmcc-msft and team.

//cc: @amnguye, @schaabs

Reply:
I'm fine with this change right now.

Actually I'm actually working on fixing this right now. I'm actually trying to make a bigger change that way we don't have to keep swapping back and forth as often.

This is mostly here because our tests have a dependency on whenever we update our `ServiceVersion` and the main branch is in development.

Reply:
Agreed, we actually need this change before we branch for STG 95.